sourced/ml/utils/spark.py

```diff
@@ -68,6 +73,18 @@ def create_spark(session_name,
     session.sparkContext.setLogLevel(spark_log_level)
     # Hide py4j verbose logging (It appears in travis mostly)
     logging.getLogger("py4j").setLevel(logging.WARNING)
+    if dep_zip:
+        zip_path = os.path.expanduser('~/.cache/source{d}/ml/sourced.zip')
```

Comment:
We use double quotes in our projects, see https://github.com/src-d/guide/blob/master/engineering/python-style-guide.md Rationale: relax transitions from Go to Python and back